jdaviz/app.py

```diff
@@ -465,14 +470,101 @@ def _color_to_level(color):
     def _on_layers_changed(self, msg):
         if hasattr(msg, 'data'):
             layer_name = msg.data.label
+            is_wcs_only = msg.data.meta.get(self._wcs_only_label, False)
         elif hasattr(msg, 'subset'):
             layer_name = msg.subset.label
+            is_wcs_only = False
         else:
             raise NotImplementedError(f"cannot recognize new layer from {msg}")
 
+        wcs_only_refdata_icon = ''  # blank - might be replaced with custom icon in the future
+        # any changes here should also be manually reflected in orientation.vue
+        orientation_icons = {'Default orientation': 'mdi-image-outline',
```

Comment:
I think this is showing up unintentionally in Plot Options. Run ImvizDitheredExample notebook, load both images, link by WCS, open up Plot Options and see this:

![Screenshot 2024-01-18 144130](https://github.com/spacetelescope/jdaviz/assets/2090236/d8cb59d9-4797-42f3-ba3f-27bd49ee4971)

Reply:
This is when I also have N-E left orientation. Why are the orientation fake WCS layers showing up in Plot Options?

![Screenshot 2024-01-18 144510](https://github.com/spacetelescope/jdaviz/assets/2090236/69c471cd-f973-49c2-a274-6dc567b1a042)

Reply:
Your latest commit seems to have fixed this. Thanks!